.jshintrc

```diff
@@ -111,6 +111,9 @@
         "afterEach"             : false,
         "spyOn"                 : false,
         "xdescribe"             : false,
-        "xit"                   : false
+        "xit"                   : false,
+
+        // objects
+        "Map"                   : false
```

Comment:
You have the wrong npm dependency versions for 2.1.x, if you are hitting this issue.